electron/index.js

```diff
@@ -113,7 +113,9 @@ class CapacitorSplashScreen {
     `;
 
     this.mainWindowRef.on('closed', () => {
-      this.splashWindow.close();
+      if (this.splashWindow && !this.splashWindow.isDestroyed) { 
```

Comment:
`isDestroyed` is a function, not a property. This condition will always return false.

Reply:
Fixed with latest commit. Thanks, @benjcallaghan!